skimage/restoration/inpaint.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import print_function, division
+
+import numpy as np
+import skimage
+from scipy import sparse
+from scipy.sparse.linalg import spsolve
+from scipy.ndimage.filters import laplace
+
+
+def inpaint_biharmonic(img, mask, multichannel=False):
+    """Inpaint masked points in image with biharmonic equations.
+
+    Parameters
+    ----------
+    img : nD{+color channel} np.ndarray
```

Comment:
We usually write something along the lines of "array, shape (M, N, [..., P[, 3]])" (or C for the last dimension, if any number of channels is supported)